setup.py

```diff
@@ -18,12 +24,15 @@
         "Topic :: Scientific/Engineering :: Information Analysis",
```

Comment:
Add Python 3.9 as in openfisca-core to avoid compatibility errors?
```suggestion
        'Programming Language :: Python :: 3.9',
        "Topic :: Scientific/Engineering :: Information Analysis",
```